model/src/main/kotlin/io/spine/validation/bound/KotlinNumericBound.kt

```diff
@@ -121,7 +121,7 @@ internal fun KotlinNumericBound.toProto(): NumericBound {
 }
 
 /**
- * Parses the given string [value] to a [KotlinNumericBound].
+ * Parses the given raw [bound] string to a [KotlinNumericBound].
```

Comment:
I cannot comment on the above code, so I do it here. What's the value of having the `Kotlin` prefix of this class name? We are already under the `kotlin` directory. Everything around is written in Kotlin. The class is `internal`.

Why do we need to stress on the fact that this class uses unsigned types from Kotlin? Are we shy about it, or what?

Reply:
>> What's the value of having the Kotlin prefix of this class name?

Because it is a one-to-one copy of Protobuf-generated `NumericBound`, but with Kotlin types used to represent a number (instead of primitives). This simplifies parsing and comparison (the latter is needed further to check the relation of the both range bounds).

Reply:
Can we have it as `KNumericBound` then? As a last resort.